_posts/2019-02-07-checking-the-status-of-your-api-submitted-deal.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title:  "Checking the Status of Your API Submitted Deal"
+date:   2019-02-07 4:00:00 -0400
+categories: api
+author: Kyle Chan
+---
+
+*Now that you have submitted your first deal via our [Lead Submission Endpoint](http://tech.forwardfinancing.com/api/2017/03/07/using-our-lead-submissions-api.html),
+you're probably wondering how it's doing.*
+
+## Getting Started
+The only things you'll need are 
+- The **lead ID** you received when you first submitted your deal
+- Your **API key**
+
+#### Authentication
+
+You will need to contact us to get your **API key**. The API key should be included
+in your request headers such that the request header key is `api_key` and the
+request header value for that key is your **API key**. If you omit the api_key or
+provide an incorrect one, the HTTP status code of the response will be 401
+Unauthorized.
+
+#### HTTPS
+
+Since your API requests will contain sensitive personal information about our
+customers, you must use SSL (HTTPS). To discourage insecure API requests,
+the response code will be 403 Forbidden for all requests made over HTTP.
+
+#### Rate Limiting
+
+We impose a rate limit per user of 1 request per second. If you exceed this rate
+of requests, the response code will be 429 Too Many Requests. If you need to
+exceed this limit for some reason, please let us know!
+
+## The Deal Status Endpoint
+
+You will be able to check your deal status with a GET request to the `/v1/deal_status` 
+endpoint with your **lead ID**.
+For example, the GET url might be
+
+`https://api-staging.forwardfinancing.com/v1/deal_status/<LEAD_ID>`
+
+### Request
+
+The only thing you'll need in the request header is your **API key**!
+
+```
+api_key: YOUR_API_KEY_HERE
+```
+
+### Response
+The possible response body fields are 
+- stage
+- maxApproval
+- maxPayments
+- offerLink
+- declineDrivers
+- declineNotes
+- missingInfo
+
+Your response will contain different fields depending on the status of your deal.
+All responses will contain the **stage** of your deal.
+
+|**Deal Status**|stage  | maxApproval   | maxPayments  | offerLink  | declineDrivers  | declineNotes  | missingInfo |
```

Comment:
Does anyone know a better way of doing this? Might be a bit hard to read because there is no differentiation in the rows like there is w/ GH markdown.

Reply:
What do you mean here?

Reply:
It comes out looking like this: 
![image](https://user-images.githubusercontent.com/19920280/52574448-55683280-2dea-11e9-9a6a-c1b05797cf2c.png)

Is that illegible? There's definitely some styling going on because github renders tables like [this](https://help.github.com/articles/organizing-information-with-tables/) with borders within and along the table and different colored rows.

Reply:
Yeah thats not super easy to read but I don't think its illegible. I bet there is a way to change the default styling either via the markdown itself or by changing css/config somewhere else in this repo